src/py/flwr/driver/driver_client_manager.py

```diff
@@ -35,6 +35,9 @@ class DriverClientManager(ClientManager):
 
     def __init__(self, driver: Driver) -> None:
         self.driver = driver
+        self.workload_id = driver.create_workload(
+            driver_pb2.CreateWorkloadRequest()
+        ).workload_id
```

Comment:
Let's do this lazily when one of the other methods is called

Reply:
sure